pkg/acsengine/defaults.go

```diff
@@ -178,6 +178,21 @@ var (
 			},
 		},
 	}
+
+	// DefaultReschedulerAddonsConfig is the default rescheduler Kubernetes addon Config
+	DefaultReschedulerAddonsConfig = api.KubernetesAddon{
+		Name:    DefaultReschedulerAddonName,
+		Enabled: pointerToBool(true),
+		Containers: []api.KubernetesContainerSpec{
+			{
+				Name:           DefaultReschedulerAddonName,
+				CPURequests:    "300m",
+				MemoryRequests: "150Mi",
+				CPULimits:      "300m",
+				MemoryLimits:   "150Mi",
```

Comment:
^^ need to discuss what default limits to use for rescheduler

Reply:
We should copy what's here:

https://github.com/kubernetes/kubernetes/blob/master/cluster/saltbase/salt/rescheduler/rescheduler.manifest

(limits can copy requests)